mat44xmat44_sse.c

```diff
@@ -0,0 +1,127 @@
+#include <immintrin.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/time.h>
+
+#define N 4
+
+int
+main() {
+
+    // allocate memory for array
+    int are_equal = 1;
+    int count = 0;
+    float(*A)[N];
```

Comment:
same comments as on the other file, float A[4][4] with the aligned attribute are enough here.